# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/test/jsmodules/chrome.toml
# Commit: 587cbe9d2aaa
# Full Hash: 587cbe9d2aaae70b2eedbf5801a228bc9421a38d
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-09-26 06:07 +0000
# Regressor Bug: 1968890
# File Overlap Count: 3
# Overlapping Files: dom/base/test/jsmodules/chrome.toml, dom/base/test/jsmodules/import_404.mjs, dom/base/test/jsmodules/test_dynamicImportErrorMessage.html
# Description:
#   Bug 1968890 - Part 9: Add test for dynamic import. r=jonco
# ==============================================================================

diff -r fe82822fa4a1 -r 587cbe9d2aaa dom/base/test/jsmodules/chrome.toml
--- a/dom/base/test/jsmodules/chrome.toml	Fri Sep 26 06:07:45 2025 +0000
+++ b/dom/base/test/jsmodules/chrome.toml	Fri Sep 26 06:07:45 2025 +0000
@@ -1,6 +1,7 @@
 [DEFAULT]
 support-files = [
   "ambiguous_export.mjs",
+  "import_404.mjs",
   "import_ambiguous.mjs",
   "import_ambiguous_indirect_export.mjs",
   "import_ambiguous_export.mjs",